tests/test_lambdas/test_broker.py

```diff
@@ -0,0 +1,94 @@
+from lbz.exceptions import NotFound
+from lbz.lambdas import LambdaBroker, LambdaResponse, LambdaResult, lambda_ok_response
+
+
+def simple_func(*_event: dict) -> LambdaResponse:
+    return lambda_ok_response()
+
+
+class TestEventBroker:
+    def test_broker_works_properly_when_data_is_provided(self) -> None:
+        def func(_event: dict) -> LambdaResponse:
+            return lambda_ok_response({"some": "data"})
+
+        mapper = {"x": func}
+        event = {"op": "x", "data": {"y": 1}}
+
+        resp = LambdaBroker(mapper, event).react()
+
+        assert resp == {
+            "result": LambdaResult.OK,
+            "data": {"some": "data"},
+        }
+
+    def test_broker_works_properly_when_no_data_is_provided(self) -> None:
+        mapper = {"x": simple_func}
```

Comment:
Also, you need to create a handler that does not accept `data` at all - `simple_func` doesn't care.